src/modules/file-preview-card.marko

```diff
@@ -8,22 +8,20 @@
     <h3>Uploading File Preview Card</h3>
     <div class="demo">
         <div class="demo__inner">
-            <div style="height: 168px; width: 168px;">
+            <div style="width: 168px">
```

Comment:
Is this width needed? I know it was here before but shouldn't we have sizes based on the grid for this?

Reply:
This is an arbitrary width just for the example. The container of the component is what decides its width